pages/article/[articleLink].js

```diff
@@ -0,0 +1,157 @@
+      setArticle(await getArticleDetail(articleLink));
+    };
+
+    if (articleLink) {
+      handleFetching();
+    }
+  }, [articleLink]);
+
+  if (!article) {
+    return <Loader />;
+  }
+
+  const {
+    id,
+    title,
+    content,
+    link,
+    featuredImage,
+    details,
+    keywords,
+    portals,
+    altForFeaturedImage,
+  } = article;
+
+  return (
+    <Layout>
+      <div key={id}>
+        <Link href={`/article/edit/${link}`}>
+          <a className="block-inline text-right text-blue-600 underline">
+            <h2>Editovat článek</h2>
+          </a>
+        </Link>
+
+        <h1 className="headline--1 underline-large mb-5">{title}</h1>
+        <div className="mt-5 flex flex-col-reverse md:flex-row">
+          <div className="md:mr-[5%] md:w-[70%]">
+            <p>{decodeURIComponent(content)}</p>
+          </div>
+
+          <aside className="mt-8 w-full md:m-0 md:w-1/4" aria-label="sidebar">
+            {featuredImage && (
+              <figure>
+                <div className="relative block h-[300px] w-full">
+                  <Image
+                    src={decodeURIComponent(featuredImage)}
+                    layout="fill"
+                    className="relative block"
+                    objectFit="contain"
+                  />
+                </div>
+                {altForFeaturedImage && (
+                  <figcaption className="pb-5 text-center text-sm text-dark-300">
+                    {decodeURIComponent(altForFeaturedImage)}
+                  </figcaption>
+                )}
+              </figure>
+            )}
+            {details && (
+              <div>
+                <div
+                  onClick={openDetailsHandler}
+                  className="flex cursor-pointer justify-between bg-accent-200 p-3 font-bold md:hidden"
+                >
+                  <h2>
+                    {' '}
+                    {!isDetailsOpen ? 'Zobrazit detaily' : 'Skrýt detaily'}
+                  </h2>
+                  <Arrow
+                    className={classNames(
+                      { 'rotate-180': !isDetailsOpen },
+                      'w-4',
+                    )}
+                  />
+                </div>
+
+                <section
+                  className={classNames(
+                    { hidden: !isDetailsOpen },
+                    'pb-10 md:block',
+                  )}
+                >
+                  <h3 className="py-3 font-yrsa text-lg font-medium uppercase tracking-widest shadow-neutral-900">
+                    Detaily
+                  </h3>
+                  {Object.entries(details).map((item) => (
+                    <div key={item[0]} className="bg-stone-100 p-2">
+                      <h4 className="font-bold tracking-wide text-accent-600">
+                        {item[0]}
+                      </h4>
+                      <p className="text-dark-600">{item[1]}</p>
+                    </div>
+                  ))}
```

Comment:
```suggestion
                  {Object.entries(details).map(([articleId, name]) => (
                    <div key={articleId} className="bg-stone-100 p-2">
                      <h4 className="font-bold tracking-wide text-accent-600">
                        {articleId}
                      </h4>
                      <p className="text-dark-600">{name}</p>
                    </div>
                  ))}
```

Used some random names, not sure what it actually has inside :D. Please rename it to suit your needs.